Sources/BadondeCore/BurghCommand.swift

```diff
@@ -117,23 +117,24 @@ class BurghCommand: Command {
 
 		Logger.step("Diffing base and target for label derivation")
 		let diffs = try [Diff](baseBranch: Branch(name: pullRequestBaseBranch, source: .remote(remote)), targetBranch: currentBranch)
-
+		let addedFiles = diffs.compactMap { $0.addedFilePath }
+		
```

Comment:
<!--
  1 failure:  Lines should not ...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/BadondeCore/BurghCommand.swift;
  Line: 121;
-->
- :no_entry_sign: Lines should not have trailing whitespace. (`trailing_whitespace`)